src/map/pc.c

```diff
@@ -4879,11 +4879,6 @@ int pc_isUseitem(struct map_session_data *sd,int n)
 		return 0; // You cannot use this item while sitting.
 	}
 
-	if (sd->state.storage_flag != STORAGE_FLAG_CLOSED && item->type != IT_CASH) {
```

Comment:
if (sd->state.storage_flag != STORAGE_FLAG_CLOSED && (battle_config.storage_use_item || item->type != IT_CASH))
Above condition suites better, and remove modification of pc_useitem

Reply:
About your 1st comment, my changes here are correct, the pc_isUseItem somehow works but the behavior isnt correct. In kRO, if you open a cash box, it will still show the warning message, but it will not open. This behavior works in pc_useitem, while in pc_isUseItem, you cannot open the box at first place.